src/script/components/CellsGlobalView/common/useCellsStore/useCellsStore.ts

```diff
@@ -17,19 +17,22 @@
  *
  */
 
+import {RestPagination} from 'cells-sdk-ts';
 import {create} from 'zustand';
 
 import {CellFile} from '../cellFile/cellFile';
 
-type Status = 'idle' | 'loading' | 'success' | 'error';
+export type Status = 'idle' | 'loading' | 'success' | 'error';
 
 interface CellsState {
   files: CellFile[];
   status: Status;
+  pagination: RestPagination | null;
```

Comment:
I'm not a big fan of injecting the whole pagination object as is from the cells-sdk. It kinda couples us with the interface, the naming convension is different (like "TotalPages" instead of "totalPages")...

What I'd do it to have our own type and map incoming pagination from API to match our own.

```suggestion
  pagination: {
    totalPages: number;
    ...
  } | null;
```

Reply:
created a new CellPagination model similarly to CellFile